tool/site/client-react/src/components/prescription-summary/prescriptionLevelDetails.tsx

```diff
@@ -0,0 +1,135 @@
+import {SummaryList} from "nhsuk-react-components"
+import * as React from "react"
+import {CommunicationRequest, CommunicationRequestPayload, MedicationRequest} from "fhir/r4"
+import {formatDate} from "../../formatters/dates"
+import {
+  getNumberOfRepeatsAllowedExtension,
+  getNumberOfRepeatsIssuedExtension, getPerformerSiteTypeExtension,
+  getRepeatInformationExtension
+} from "../../fhir/customExtensions"
+import {brLineFragments} from "./brLineFragments";
+
+export function createPrescriptionLevelDetails(medicationRequest: MedicationRequest, communicationRequests?: Array<CommunicationRequest>): PrescriptionLevelDetailsProps {
+  const prescriptionId = medicationRequest.groupIdentifier.value
+
+  const repeatExtension = getRepeatInformationExtension(medicationRequest.extension)
+  const repeatsIssuedExtension = getNumberOfRepeatsIssuedExtension(repeatExtension.extension)
+  const repeatIssued = repeatsIssuedExtension.valueInteger
+  const repeatsAllowedExtension = getNumberOfRepeatsAllowedExtension(repeatExtension.extension)
+  const repeatAllowed = repeatsAllowedExtension.valueInteger
+
+  const authoredOn = formatDate()
+  const startDate = formatDate(medicationRequest.dispenseRequest.validityPeriod?.start) ?? formatDate()
```

Comment:
Don't need the `?? formatDate()`, although it's maybe a bit unclear that it will return the current date if there's no `validityPeriod.start`

Reply:
Maybe we should have a separate function for the current date and have something like:
```
const validityPeriodStart = medicationRequest.dispenseRequest.validityPeriod?.start
const startDate = validityPeriodStart ? formatDate(validityPeriodStart) : formatCurrentDate()
```